src/components/Patient/PatientDetailsTab/Demography.tsx

```
@@ -0,0 +1,460 @@
+              {formatPatientAge(patientData, true)})
+            </>
+          ),
+        },
+        {
+          label: t("sex"),
+          value: patientGender,
+        },
+        <EmergencyContact
+          number={patientData.emergency_phone_number}
+          name={patientData.name}
+        />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)